README.md

```diff
@@ -0,0 +1,93 @@
+[![Lint](https://github.com/Mean-Finance/subsidies/actions/workflows/lint.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/subsidies/actions/workflows/lint.yml)
+[![Tests](https://github.com/Mean-Finance/subsidies/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/subsidies/actions/workflows/tests.yml)
+[![Slither Analysis](https://github.com/Mean-Finance/subsidies/actions/workflows/slither.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/subsidies/actions/workflows/slither.yml)
+
+# Ongoing airdrops / campaigns
+
+This repository holds the [OngoingAirdrops.sol](./solidity/contracts/OngoingAirdrops.sol) smart contract that is implemented to be able to handle multiple ongoing airdrops, also referred to as **campaigns**.
+
+## Multiple campaigns
+
+Let's say a protocol wants to do a one-time referral reward in USDC for all users that have done some action. Additionally, this protocol wants to do a one-time airdrop of some loyalty token.
+
+Instead of having to deploy multiple airdrop claiming smart contracts and making more difficult the task to maintain subgraphs, frontends, etc. they can now create both campaigns on this contract. One might be called `Best swappers` and the other one `Loyalty Program Q1`.
+
+## Ongoing campaigns
+
+Now the protocol wants to implement an on-going (not a one time) referral campaign that for one reason or another can not be calculated and / or distributed directly / instantly on-chain. Now, the protocol could still want the users to claim this off-chain calculated referral on-chain.
```

Comment:
```suggestion
Another potential use case would be if a protocol wants to implement an on-going (not a one time) referral campaign that for one reason or another cannot be calculated and / or distributed directly on-chain. Instead of having multiple claim contracts, this `OngoingAirdrops` contract can be used instead
```